src/library/scala/collection/immutable/Stream.scala

```diff
@@ -933,13 +933,34 @@ self =>
    * // produces: "1, 2, 3, 4, 5, 6"
    * }}}
    */
-  override def distinct: Stream[A] = {
+  override def distinct: Stream[A] = distinctBy(identity)
+
+  /** Builds a new stream from this stream in which elements with duplicate
```

Comment:
The first line always shows in ScalaDoc, and is meant to be a short self-contained explanation of the functionality. Please shorten the line to something like this:

    Builds a new stream in which elements mapped to the same value with function `f` are de-duplicated.

The, separate paragraph can describe semantics in more detail, e.g. the first element is always kept, etc.